Code/Framework/AzCore/AzCore/Settings/SettingsRegistryMergeUtils.cpp

```diff
@@ -337,16 +335,17 @@ namespace AZ::SettingsRegistryMergeUtils
             return engineRoot;
         }
 
-        // Step 3 locate the project root and attempt to find the engine root using the registered engine
-        // for the project in the project.json file
+        // Step 3 locate the project root and attempt to find the engine using the engine_path
+        // in user/project_settings.json or find the first registered engine with the same name
```

Comment:
The project_settings.json naming scheme needs to be updated.
```suggestion
        // in <project-root>/user/project.json or find the first registered engine with the same name
```